apps/experiments/tables.py

```diff
@@ -167,6 +167,9 @@ class ExperimentVersionsTable(tables.Table):
         {% endif %}""",
         verbose_name="Default Version",
     )
+    details = columns.TemplateColumn(
```

Comment:
Super nit : Thoughts on calling it something like "view" rather? I can't articulate exactly why, but if I were to click on a button that says "details" and see some actions where I can for instance archive the version of the experiment, my brain stop and asks "wait, what just happened". If this is not the case with you, fine to leave it as-is :smile: